batchflow/models/tf/mobilenet.py

```diff
@@ -310,24 +298,37 @@ def head(cls, inputs, se_block=None, name='head', **kwargs):
         name : str
             scope name
         se_block : dict, optional
-            If not None, it must be a dict containing :meth:`~TFModel.se_block` params
+            params for :meth:`~TFModel.se_block`
```

Comment:
It is confusing to see the head with hardcoded parameters from version 3 in version 2.